README.md

```diff
@@ -256,33 +258,40 @@ const amioWebhookRouter = require('path/to/amio-webhook-router.js')
 app.use('/', amioWebhookRouter)
 ```
 
+### Webhooks - methods
+
+amioWebhookRouter.* | Description
+-|-
+`setSecret(channelId, secret)` | Pairs a channelId and a secret. Setting channelId twice will override the previous secret.
```

Comment:
Pairs a `channelId` with a `secret`. Setting `secret` for the same `channelId` twice will override the previous value.